ipa-core/src/secret_sharing/into_shares.rs

```diff
@@ -41,7 +41,7 @@ where
     }
 }
 
-#[cfg(all(test, unit_test))]
+#[cfg(test)]
```

Comment:
for my own understanding, what is the difference between `test` and `unit_test`?

Reply:
mainly `unit_test` includes `descriptive_gate`: https://github.com/private-attribution/ipa/blob/main/ipa-core/build.rs#L48 which prevents these trait implementations from being used in compact gate tests